Sources/Conformance/test_messages_proto3.pb.swift

```diff
@@ -1673,480 +1150,987 @@ struct ProtobufTestMessages_Proto3_TestAllTypes: SwiftProtobuf.Proto3Message, Sw
       }
-      if storage._fieldName10 != 0 {
-        try visitor.visitSingularInt32Field(value: storage._fieldName10, fieldNumber: 410)
+      if _storage._fieldName10 != 0 {
+        try visitor.visitSingularInt32Field(value: _storage._fieldName10, fieldNumber: 410)
       }
-      if storage._fieldName11 != 0 {
-        try visitor.visitSingularInt32Field(value: storage._fieldName11, fieldNumber: 411)
+      if _storage._fieldName11 != 0 {
+        try visitor.visitSingularInt32Field(value: _storage._fieldName11, fieldNumber: 411)
       }
-      if storage._fieldName12 != 0 {
-        try visitor.visitSingularInt32Field(value: storage._fieldName12, fieldNumber: 412)
+      if _storage._fieldName12 != 0 {
+        try visitor.visitSingularInt32Field(value: _storage._fieldName12, fieldNumber: 412)
       }
-      if storage.__FieldName13 != 0 {
-        try visitor.visitSingularInt32Field(value: storage.__FieldName13, fieldNumber: 413)
+      if _storage.__FieldName13 != 0 {
+        try visitor.visitSingularInt32Field(value: _storage.__FieldName13, fieldNumber: 413)
       }
-      if storage.__FieldName14 != 0 {
-        try visitor.visitSingularInt32Field(value: storage.__FieldName14, fieldNumber: 414)
+      if _storage.__FieldName14 != 0 {
+        try visitor.visitSingularInt32Field(value: _storage.__FieldName14, fieldNumber: 414)
       }
-      if storage._field_Name15 != 0 {
-        try visitor.visitSingularInt32Field(value: storage._field_Name15, fieldNumber: 415)
+      if _storage._field_Name15 != 0 {
+        try visitor.visitSingularInt32Field(value: _storage._field_Name15, fieldNumber: 415)
       }
-      if storage._field_Name16 != 0 {
-        try visitor.visitSingularInt32Field(value: storage._field_Name16, fieldNumber: 416)
+      if _storage._field_Name16 != 0 {
+        try visitor.visitSingularInt32Field(value: _storage._field_Name16, fieldNumber: 416)
       }
-      if storage._fieldName17__ != 0 {
-        try visitor.visitSingularInt32Field(value: storage._fieldName17__, fieldNumber: 417)
+      if _storage._fieldName17__ != 0 {
+        try visitor.visitSingularInt32Field(value: _storage._fieldName17__, fieldNumber: 417)
       }
-      if storage._fieldName18__ != 0 {
-        try visitor.visitSingularInt32Field(value: storage._fieldName18__, fieldNumber: 418)
+      if _storage._fieldName18__ != 0 {
+        try visitor.visitSingularInt32Field(value: _storage._fieldName18__, fieldNumber: 418)
       }
     }
   }
 
   func _protoc_generated_isEqualTo(other: ProtobufTestMessages_Proto3_TestAllTypes) -> Bool {
-    return _storage === other._storage || _storage.isEqualTo(other: other._storage)
-  }
-
-  private mutating func _uniqueStorage() -> _StorageClass {
-    if !isKnownUniquelyReferenced(&_storage) {
-      _storage = _storage.copy()
+    return withExtendedLifetime((_storage, other._storage)) { (_storage, other_storage) in
```

Comment:
It's no longer possible to short-circuit on storage identity because unknown fields and extensions aren't stored there. So, knowing that two messages have the same storage instance doesn't tell you anything about those, and knowing that two messages have different storage instances doesn't let you assume that the contents aren't different because they could have diverged and then been set back to match.